CAVEATS.md

```diff
@@ -0,0 +1,52 @@
+# Platform specific notes
+
+## Windows
+
+### Spectest requirements
+
+You need a recent version of either Strawberry Perl or ActiveState Perl.
+
+If you are working with ActiveState Perl you will need the Mingw gcc
+compiler.
```

Comment:
This is a curiosity; I've tended to use ActiveState Perl + MSVC and it worked fine (still do that on the occasions I do a Windows build). Is the point that you'll need a C compiler (whereas Strawberry Perl bundles one), and it's being maybe more specific than it need be in saying which one?

Reply:
Guessing this came from the install text. I also use msvc- we can update the text to refer to both

Reply:
@coke guessed it right.

I only do a few builds on Window once in a long while. So, it's for somebody with better expertise to fix this paragraph.